polkadot/xcm/pallet-xcm/src/tests/assets_transfer.rs

```diff
@@ -907,7 +869,7 @@ fn destination_asset_reserve_and_destination_fee_reserve_call<Call>(
 		assert_eq!(result, expected_result);
 		if expected_result.is_err() {
 			// short-circuit here for tests where we expect failure
-			return
+			return;
```

Comment:
Please no unrelated changes.

Reply:
@ggwpez  Resolved in this commit https://github.com/paritytech/polkadot-sdk/pull/4248/commits/9d552711d9a2c95be9478786bf5665a4113be639. Thank you for feedback!